cypress/integration/userCanViewTimeHistory.feature.js

```diff
@@ -0,0 +1,26 @@
+/// <reference types="Cypress" />
+
+describe("User can view history of time recordings", () => {
+	it("successfully displays a list", () => {
+		cy.server();
+		cy.route({
+			method: 'GET',
+			url: 'https://demo-stable.kimai.org/api/timesheets',
+			response: 'fixtures:save_data.json',
+			headers: {
+				"X-AUTH-USER": "susan_super",
+				"X-AUTH-TOKEN": "api_kitten"
+			}
+		})
+		cy.visit("http://localhost:3000")
+		cy.get('input[type=name]').type('anna_admin')
+		cy.get('input[type=password]').type('api_kitten')
+		cy.get('button[type=submit]').click()
+		cy.get('input[name="begin"]').type("07:00")
+		cy.get('input[name="end"]').type("08:00")
+		cy.get('button[name="create"]').click()
+		cy.contains('Your time was saved')
+		cy.contains('07:00')
+		cy.contains('08:00')
+	})
+});
```

Comment:
I can't see how this test relates to the current user story. Please explain

Reply:
we would like to show the time recordings displayed on the page after we submitted them. we can elaborate on the test once we have info displayed
![image](https://user-images.githubusercontent.com/45356724/55708579-8c238900-59e6-11e9-956c-4e617289f7b4.png)

Reply:
we got clarity on this, agree. will rectify to only focus on testing for fetching the info